From 343cb5a8b11a1dfdf4262fce326113e889f13496 Mon Sep 17 00:00:00 2001
From: Anders Darander <anders@chargestorm.se>
Date: Sat, 9 Nov 2013 23:05:05 +0100
Subject: [PATCH] interpreter/vm.c: include uaccess.h

otherwise functions, such as copy_from_user, is implicitly defined.

Signed-off-by: Anders Darander <anders@chargestorm.se>
---
 interpreter/vm.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/interpreter/vm.c b/interpreter/vm.c
index bc7b951..f7cb794 100644
--- a/interpreter/vm.c
+++ b/interpreter/vm.c
@@ -28,6 +28,7 @@
 #include <linux/signal.h>
 #include <linux/sched.h>
 #include "../include/ktap.h"
+#include <linux/uaccess.h>
 
 #define KTAP_MINSTACK 20
 
-- 
1.8.4.2

